sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -417,6 +481,17 @@ static struct snd_soc_dai_link_component max98357a_component[] = {
 	}
 };
 
+static struct snd_soc_dai_link_component rt1015_components[] = {
+	{
+		.name = "i2c-10EC1015:00",
+		.dai_name = "rt1015-aif",
+	},
+	{
+		.name = "i2c-10EC1015:01",
+		.dai_name = "rt1015-aif",
+	},
+};
```

Comment:
@yongzhi1 can we please use the macro SND_SOC_DAILINK_DEF here for defining the codec components?

Reply:
Principally yes, I also tried to match the original style here :)

Reply:
@ranj063 @yongzhi1 That's probably better done in a separate patch if you need to touch existing definitions.

Reply:
ack!